### DISCUSSION TITLE
Deep handling in /public directory

### DISCUSSION THREAD

**Discussion by Abjuk** [vercel/vercel#4298] [General] [answered]
For verify ssl certificate I should use this path: /.well-known/pki-validation/6377BE897262BF21456459DF909610333642.txt

If I put this file to /public/6377BE897262BF21456459DF909610333642.txt -> all ok
If I use /public/.well-known/pki-validation/6377BE89726456456F21C9DF909610333642.txt - I got 404 error.

I can't change this patch because it rule from SSL Certificate Center
Could you please help me.

**Comment by coetry** [comment, marked as answer]
Hi @Bielykh-Artem We automatically generate SSL certificates for all of our customers and if you'd like to use a custom SSL certificate, we have a process to make that available for our Enterprise customers with a special contract. If you are interested in that, please send an email to enterprise@vercel.com
